### PR TITLE
Add `current` command to retrieve the current Node version

### PR DESCRIPTION
**Pull request by vladimyr**
Display currently activated Node version.

### CODE REVIEW

**Comment by Schniz** [comment]
Cool! can we also add a small feature test? 😄

**Comment by vladimyr** [comment]
> Cool! can we also add a small feature test? 😄

Shell or reason?

**Comment by Schniz** [comment]
I like using shell because it shows how the user will interact with your feature. But you can use Reason too, as long as we're sure that I won't break your feature in the future 😄

**Comment by vladimyr** [comment]
Feature tests added 🎉

File: feature_tests/current/run.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -e
+
+PATH="$(pwd)":$PATH # simulating a custom `node`
```
Comment (Schniz) [inline feature_tests/current/run.sh:5]: I'm not sure how it simulates a custom node

File: executable/Current.re [hunk 2]
```diff
@@ -0,0 +1,11 @@
+open Fnm;
+
+let run = () => {
+  let%lwt currentVersion = Versions.getCurrentVersion();
+  switch (currentVersion) {
+  | Some({name}) => Console.log(name)
+  | _ => Console.log("none")
+  };
```
Comment (Schniz) [inline executable/Current.re:8]: So slick

File: feature_tests/current/run.sh (hunk 1 above)
Comment (Schniz) [reply feature_tests/current/run.sh:5]: but I'm merging this anyway, we can remove it later

Comment (vladimyr) [reply feature_tests/current/run.sh:5]: > I'm not sure how it simulates a custom node

You are absolutely right. Simulating custom `node` here isn't needed and this doesn't do that anyway 🤦

File: executable/Current.re (hunk 2 above)
Comment (vladimyr) [reply executable/Current.re:8]: I hope so, this is the first thing I'm doing in OCaml/Reason, ignoring obligatory Hello World, and definitely isn't much more advanced 😁

**Pull request merged**